pkg/providers/tinkerbell/stack/stack.go

```diff
@@ -298,6 +301,13 @@ func (s *Installer) getBootsEnv(bundle releasev1alpha1.TinkerbellStackBundle, ti
 			bootsEnv["REGISTRY_PASSWORD"] = password
 		}
 	}
+	if s.proxyConfig != nil {
+		noProxy := strings.Join(s.proxyConfig.NoProxy, ",")
+		extraKernelArgs = fmt.Sprintf("%s HTTP_PROXY=%s HTTPS_PROXY=%s NO_PROXY=%s", extraKernelArgs, s.proxyConfig.HttpProxy, s.proxyConfig.HttpsProxy, noProxy)
+		bootsEnv["HTTP_PROXY"] = s.proxyConfig.HttpProxy
+		bootsEnv["HTTPS_PROXY"] = s.proxyConfig.HttpsProxy
+		bootsEnv["NO_PROXY"] = noProxy
```

Comment:
For what does Boots use these values?

Reply:
Thanks for pointing this out. I have removed the block. Not needed anymore.